defaults/main.yml

```diff
@@ -34,7 +34,8 @@ php_pecl_modules:
 php_package_state: "present"
 php_install_browscap_ini: true
 php_symlink_scl_bins: true
-php_root: "/opt/remi/{{ php_prefix }}/root"
+php_remi_root: "/opt/remi/"
```

Comment:
All paths should omit the trailing '/'.  I prob missed this in a previous review.